docs/java-agent-injector.md

```diff
@@ -111,6 +117,73 @@ volumes:
     name: java-agent-configmap-volume
 ```
 
+### Use SwAgent CR to overlay default agent configuration
+
+The injector will read the SwAgent CR when pods creating.
+
+SwAgent CRD basic structure is like:
+
+```yaml
+apiVersion: operator.skywalking.apache.org/v1alpha1
+kind: SwAgent
+metadata:
+  name: swagent
+  namespace: skywalking-system
+spec:
+  containerMatcher: '.*'
```

Comment:
The default value should be empty. If users don't specific this field, all containers would get injected.